src/components/Navbar/Account.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+
+interface NavbarAccount {
+  isUserSignedIn?: boolean
+}
+
+const NavbarAccount = ({ isUserSignedIn }: NavbarAccount) => {
+  const router = useRouter()
+
+  const handleSignInClick = () => {
+    // TODO: Open SignIn modal
+  }
+
+  const handleSignUpClick = () => {
+    router.push('/sign-up')
+  }
+
+  const handleAvatarClick = () => {
+    // TODO: Expand dropdown or sth
+  }
+
+  const renderAvatarButton = () => (
+    <button className="navbar__button-avatar" onClick={handleAvatarClick}>
+      <img src="#" alt="avatar" />
+    </button>
+  )
+
+  const renderAuthenticationButton = () => (
+    <>
+      <button className="navbar__button--sign-in" onClick={handleSignInClick}>
+        Sign In
+      </button>
+      <button className="navbar__button--sign-up" onClick={handleSignUpClick}>
+        Sign Up
+      </button>
+    </>
+  )
+
+  return (
+    <div className="navbar__account">
+      {!isUserSignedIn ? renderAvatarButton() : renderAuthenticationButton()}
+    </div>
```

Comment:
What do you think about this? 🤔
```suggestion
  <div className="navbar__account">
    {isLoggedIn
      ? renderAvatarButton()
      : renderAuthenticationButton()
    }
  </div>
```
See here: https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator

Reply:
Yeah for larger expression we do split into 3 lines like what you suggested, but in this case there are no `props`. I personally prefer one line when there is only a block of text.